### ISSUE TITLE
Remove identity template Padding, if the identity template is not provided at all

### ISSUE THREAD

**Issue by ShankyTiwari** [ShankyTiwari/mat-list-touch#17]
Hi,

As a user when I would not use the identity template, I want the it's padding to be gone. Right now its applies `padding-left` and `padding-right` 24px as shown in below image,

![image](https://user-images.githubusercontent.com/15548024/90331018-27f7d880-dfcf-11ea-8e4e-de6d0d0dc100.png)

Best,

**Comment by davidzwa** [comment]
Good point, I mentioned it in the readme as well: the styling is not 100% robust. Thanks for noticing

**Comment by ShankyTiwari** [comment]
Yeah, Cool. In the coming days, I will contribute more.
